client/wordpress-com.js

```diff
@@ -341,7 +341,14 @@ sections.push( {
 	module: 'login',
 	enableLoggedOut: true,
 	secondary: false,
-	isomorphic: true
+	isomorphic: {
+		cache: {
+			queryParams: {
+				include: [ 'client_id' ],
+				exclude: [ 'redirect_to' ],
+			}
```

Comment:
We will probably need to document this somewhere.
Here is a quick comment about what this is:
- `include` is a list of query parameters that are should render a different kind of page. The parameter key and value should be included in the cache key along with the pathname.
- `exclude` is a list of query parameters to ignore, they indicate that different values of this parameter shouldn't produce a different rendered page.